example/lib/main.dart

```diff
@@ -50,8 +49,6 @@ class MyApp extends StatelessWidget {
       ),
       home: const HomePage(),
       routes: <String, WidgetBuilder>{
-        CancellableTileProviderPage.route: (context) =>
-            const CancellableTileProviderPage(),
```

Comment:
I'm not sure that we want to use `CancellableNetworkTileProvider` everywhere, the whole point of having a separated example is to showcase its benefits compared to the default one. Moreover, the warnings are only leveraged on the web, you wouldn't have them on desktop or mobile.

Reply:
Wouldn't it be better to have a page that doesn't use the `CancellableNetworkTileProvider` or (probably better) where users have the option to toggle the usage of it?
If we don't use the `CancellableNetworkTileProvider` it floods the console with warnings when running in debug mode on the web:
![image](https://github.com/fleaflet/flutter_map/assets/34318751/7560e1fc-d8e9-4da1-84f4-e243b0349519)
This makes it harder to check for other potiential warnings and logs while testing.

This has probably been discussed before but wouldn't it make sence to embrace the tile cancellation on every platform? While it leads to delayed tile loading on the web pretty quickly it **does** impact the speed on other platforms as well by using bandwidth. If the user has a bad mobile network connection or only 3G this has a big impact too. Moreover it decreases the data usage and saves up bandwidth for the tile provider.

Back to our example application: I think it makes sense to utilise these benefits here as good as possible as showcase for "best practise". 

Would you be fine with a page that lets users toggle the usage of the `CancellableNetworkTileProvider`?

Reply:
> This has probably been discussed before but wouldn't it make sence to embrace the tile cancellation on every platform?

It's already the case but here's what @JaffaKetchup has written about that on the package:

> Although HTTP request abortion is supported on all platforms, it is especially useful on the web - and therefore recommended for web apps. This is because the web platform has a limited number of simulatous HTTP requests, and so closing the requests allows new requests to be made for new tiles.
> On other platforms, the other benefits may still occur, but may not be as visible as on the web.

---

> Would you be fine with a page that lets users toggle the usage of the CancellableNetworkTileProvider?

Sure! It would still showcase the difference with/without a cancellable tile provider so I'm fine with this.

Reply:
fixed